eng/Version.Details.xml

```diff
@@ -1,90 +1,90 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <Dependencies>
   <Source Uri="https://github.com/dotnet/dotnet" Mapping="razor" Sha="f5705c8f4c5079bba77bae8698ba1583bde0388c" BarId="269610" />
   <ProductDependencies>
-    <Dependency Name="Microsoft.Net.Compilers.Toolset" Version="5.0.0-1.25316.1">
+    <Dependency Name="Microsoft.Net.Compilers.Toolset" Version="5.0.0-1.25321.1">
```

Comment:
It doesn't look like this version has been inserted into VS main yet, so merging this PR could possibly break people, or our integration tests. It looks like if we downgrade this to `5.0.0-1.25319.11` then we get a version that has the Razor redirector and is in VS main.

The alternative is to wait a few days, or roll the dice that this will be fine. Or I guess we could kick off an integration test run for this branch?

Reply:
Scratch that, looks like `5.0.0-1.25323.2` has been inserted now, so this version should be fine.